css/style.css

```diff
@@ -411,13 +724,180 @@
     padding-right: 35px;
   }
 
+  .data__figures {
+    font-weight: 600;
+  }
+
   .test-btn {
     padding-right: 0;
   }
 
   .testdrive {
     margin-right: 60px;
   }
+
+  .info__img {
+    height: auto;
+    width: 100%;
+    max-width: 1199px;
+  }
+
+  .wrap-information {
+    max-width: 1024px;
+  }
+
+  .red-first {
+    max-width: 350px;
+    height: 744px;
+    padding: 37px 47px 0 40px;
+  }
+
+  .description__title {
+    font-size: 25px;
+  }
+
+  .drive {
+    font-size: 13px;
+    padding-bottom: 60px;
+  }
+
+  .img-inch {
+    width: 30px;
+    height: 47px;
+  }
+
+  .data__figure {
+    font-size: 32px;
+  }
+
+  .data__ins {
+    font-size: 19px;
+  }
+
+  .img-wifi {
+    width: 38px;
+    height: 28px;
+  }
+
+  .data__wireless-update {
+    font-size: 18px;
+  }
+
+  .description__bag {
+    padding-bottom: 90px;
+  }
+
+  .img-cap {
+    width: 30px;
+    height: 44px; 
+  }
+
+  .data__liters {
+    font-size: 18px;
+  }
+
+  .info-order {
+    margin: 0 -15px 0 -13px;
+  }
+
+  .info-order__text {
+    white-space: nowrap;
+    font-size: 10px;
+  }
+
+  .info-order__btn {
+    width: 132px;
+    height: 38px;
+    font-size: 12px;
+  }
+
+  .details {
+    max-width: 350px;
+    margin-top: -156px;
+    height: 744px;
+  }
+
+  .description-car {
+    padding-left: 0;
+  }
+
+  .title {
+    padding: 0px 52px 0 0;
+  }
+
+  .description__title-most {
+    font-size: 25px;
+    line-height: 28px;
+  }
+
+  .model-s {
+    font-size: 13px;
+    padding: 5px 38px 50px 0;
+  }
+
+  .rating-container {
+    padding: 0 20px 58px 0;
+  }
+
+  .rating-container__title {
+    font-size: 25px;
+    padding-bottom: 0;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)